src/controls/paginator/page-link.js

```diff
@@ -22,6 +22,7 @@ function PageLink({ className, active, onClick, children, ...rest }) {
   return (
     <li className={classnames(className, { active: active })}>
       <a
+        role="link"
```

Comment:
This is a change already made in #596 that was flagged by the `jsx-a11y` eslint plugin. If an anchor doesn't have an `href` attribute, it is not considered to have the role `link` anymore